src/sagas/RootSaga.js

```diff
@@ -66,35 +69,60 @@ export function* handleFetchProfile(action) {
 }
 
 export function* handleSaveProfile(action) {
-  const { username, accountDrafts, visibilityDrafts } = yield select(handleSaveProfileSelector);
-
   try {
+    const { username, drafts, preferences } = yield select(handleSaveProfileSelector);
+
+    const accountDrafts = keepKeys(drafts, [
```

Comment:
To save the profile, we split our drafts in two so they can be sent to the correct endpoints.